### ISSUE TITLE
click listeners do not work

### ISSUE THREAD

**Issue by thesultanster** [PierfrancescoSoffritti/android-youtube-player#105]
The click events are never fired. If I put a transparent view in front of my youtube view, the click event is fired for that transparent view, but then that transparent view steals the click event from the youtube seekbar, play, fullscreen button etc. 

Any suggestions? 

```
youTubePlayerView.setOnTouchListener(new View.OnTouchListener() {
            @Override
            public boolean onTouch(View v, MotionEvent event) {
                log.debug( "youTubePlayerView:MOTIONEVENT");
                // Get motion event type
                switch (event.getAction()) {
                    // This event is when user clicks down
                    case MotionEvent.ACTION_DOWN:
                        log.debug( "youTubePlayerView:ACTION_DOWN");
                        EventBus.getDefault().post(new VideoUIControlEvent("EXPAND_VIDEO"));
                        break;
                }
                return true;
            }

        });
```

 ```
       youTubePlayerView.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                log.debug( "youTubePlayerView:setOnClickListener");
            }
        });
```

**Comment by PierfrancescoSoffritti** [comment]
Hi, I m going to be traveling for the next couple of days. I ll get back to you when I m home this Sunday :)

**Comment by PierfrancescoSoffritti** [comment]
That's a problem related to Android, not to the library, if I understand correctly.

Just do a search on Google and learn how to prevent views from intercepting click/touch events.

**Comment by dragossusi** [comment]
did you find out how to do this?

**Comment by PierfrancescoSoffritti** [comment]
I'll double check tomorrow and let you know.

**Comment by PierfrancescoSoffritti** [comment]
At the moment I don't see any easy way to solve this without removing the click listeners from the UI controller.

**Comment by ricard-v** [comment]
Could you provide a callback when you capture the click ? That way you would not need to remove the click listeners from the UI controller.

**Comment by PierfrancescoSoffritti** [comment]
Yeah that would require adding some "non standard" click listener. I'll think about it. 
Feel free to send a pull request with your idea.

**Comment by mrkamiali** [comment]
Hi, @PierfrancescoSoffritti  is it possible if I override dispatchTouchEvent method in SixteenByNineFrameLayout class. will it be helpful to get touch listener I think so.

**Comment by mrkamiali** [comment]
@PierfrancescoSoffritti I have implemented click listener and sent you a pull request please have look.
#463

**Comment by Raunak-Dutta** [comment]
> The click events are never fired. If I put a transparent view in front of my youtube view, the click event is fired for that transparent view, but then that transparent view steals the click event from the youtube seekbar, play, fullscreen button etc.
> 
> Any suggestions?
> 
> ```
> youTubePlayerView.setOnTouchListener(new View.OnTouchListener() {
>             @Override
>             public boolean onTouch(View v, MotionEvent event) {
>                 log.debug( "youTubePlayerView:MOTIONEVENT");
>                 // Get motion event type
>                 switch (event.getAction()) {
>                     // This event is when user clicks down
>                     case MotionEvent.ACTION_DOWN:
>                         log.debug( "youTubePlayerView:ACTION_DOWN");
>                         EventBus.getDefault().post(new VideoUIControlEvent("EXPAND_VIDEO"));
>                         break;
>                 }
>                 return true;
>             }
> 
>         });
> ```
> 
> ```
>       youTubePlayerView.setOnClickListener(new View.OnClickListener() {
>            @Override
>            public void onClick(View v) {
>                log.debug( "youTubePlayerView:setOnClickListener");
>            }
>        });
> ```

Seems like this was never added, how do I add a transparent view tho?

**Comment by PierfrancescoSoffritti** [comment]
You can add a custom UI and do what you want with it. The default UI doesn't support this yet.
